fontbakery-check-ttf.py

```diff
@@ -105,6 +105,33 @@
 NAMEID_LIGHT_BACKGROUND_PALETTE = 23
 NAMEID_DARK_BACKGROUD_PALETTE = 24
 
+NAMEID_STR = {
```

Comment:
This dict serves the purpose of mapping a numeric value to its human-readable text-string for usage on logging routines where the specific nameid is only known at runtime